src/storage/adapter/cassandra.rs

```diff
@@ -0,0 +1,19 @@
+mod dek;
+
+use crate::storage::{adapter::Cassandra, errors, Config, Connection, DbState};
+
+#[async_trait::async_trait]
+impl super::DbAdapter for DbState<Cassandra> {
+    type Conn<'a> = Connection<'a>;
+    type AdapterType = Cassandra;
+
+    async fn from_config(_config: &Config) -> Self {
+        unimplemented!("Not implemented Yet")
```

Comment:
Can we include lints in `Cargo.toml`?

References:

1. Lints to include: https://github.com/juspay/hyperswitch/blob/0dba763c3a1c69fa282ffa644a12d37dc14d8f7b/Cargo.toml#L12-L38
2. Since we don't have a workspace here, refer to how we include lints in card vault: https://github.com/juspay/hyperswitch-card-vault/blob/db89f3be1e09dbf80a29828da20f5367c6d7fb1b/Cargo.toml#L106-L123

Reply:
We have this already

Reply:
Why is the CI check not failing, then?

Reply:
It looks like this being annotated with `#[async_trait::async_trait]` is causing clippy to not throw any warnings.